app/__init__.py

```diff
@@ -65,29 +57,30 @@ def linchpin_init() -> Response:
         :return : response with created workspace name,
                   id, status and code
     """
+    db_con = get_connection(DB_PATH)
```

Comment:
Can this variable db_conn be assigned globally since it is being used in every method?

Reply:
@mansikulkarni96 Yes we can. 
But choose not to because of the fact that when we have multiple requests coming in. I dont think we can handle multiple reads and writes on the same object. 
since tinydb is just a file descriptor. 
However, the current implementation needs to change when we switch over to a more standard database like MySQL or MongoDB. Where the connections might be limited.